backend/src/infra/controller/TransactionController.ts

```diff
@@ -39,4 +45,24 @@ export const TransactionController: FastifyPluginAsync<{
       return reply.status(201).send(result.payload);
     }
   );
+
+  const removeTransationToGroup = new RemoveTransactionToGroup(group);
+  fastify.delete<{ Body: RemoveTransactionInput; Params: GroupId }>(
+    "/",
```

Comment:
I think you should use `/:id` instead of passing the id through the body

So the full url would be `/group/:groupId/transaction/:id`, which is more RESTful as far as I understand it